stories/browser/src/types.ts

```diff
@@ -299,11 +299,12 @@ export type TransactionConfirmationExternalState = {
 export type TransactionConfirmationMessage = InteractiveMessage<
     InteractiveMessageType.TransactionConfirmation,
     {
-        toAddress?: string;
-        onAddressPress?: () => void | Promise<void>;
-        recipientsCount?: number;
-        totalAmount: string | React.ReactElement<any, any>;
-        fees: string | React.ReactElement<any, any>;
+        recipient?: string | React.ReactElement<any, any>;
+        onRecipientPress?: () => void | Promise<void>;
+        amount: string | React.ReactElement<any, any>;
+        action?: string | React.ReactElement<any, any>;
```

Comment:
what is action? 🤔

Reply:
<img width="306" alt="image" src="https://user-images.githubusercontent.com/36547799/164010579-f561b87c-1e8d-4b26-93a9-4eac564e8a58.png">

Reply:
Got it. Let's keep it text only